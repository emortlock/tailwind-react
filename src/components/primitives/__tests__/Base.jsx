import React from 'react'
import { shallow } from 'enzyme'

import { Base } from '..'

const setup = (testProps = {}) => {
  const props = Object.assign({}, testProps)

  const wrapper = shallow(<Base {...props}>Hello World</Base>)

  return {
    props,
    wrapper,
  }
}

describe('Base', () => {
  it('renders matching snapshot', () => {
    const { wrapper } = setup()

    expect(wrapper).toMatchSnapshot()
  })
})
