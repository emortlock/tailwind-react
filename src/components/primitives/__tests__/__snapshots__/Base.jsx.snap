// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Base renders matching snapshot 1`] = `
<div
  className=""
>
  Hello World
</div>
`;
